helpers/control.py

```diff
@@ -329,6 +330,9 @@ def stop_tempesta(self):
         tf_cfg.dbg(3, '\tStoping TempestaFW on %s' % self.host)
         cmd = '%s/scripts/tempesta.sh --stop' % self.srcdir
         self.node.run_cmd(cmd, timeout=30, err_msg=(self.err_msg % 'stop'))
+        self.oops.update()
+        if self.oops.warn_count("Oops") > 0:
+            raise Exception("Oopses happened during on Tempesta")
```

Comment:
Basically this is incorrect place to throw the exceptions. Oopses may take place after Tempesta is unloaded. This check should happen at `tearDown()`.

Reply:
fixed